webpack.config.js

```diff
@@ -79,13 +89,8 @@ const config = {
       safe: false,
       systemvars: true
     }),
+    new LodashPlugin(),
   ]
 };
 
-if (env === 'production') {
-  config.plugins.push(
-    new Uglify()
-  );
-}
-
 module.exports = config;
```

Comment:
could this be export default instead?

Reply:
That syntax isn't supported unless the js file is transpiled before running